Common/include/config_structure.hpp

```diff
@@ -1039,71 +1066,71 @@ class CConfig {
   //
   // List and Array options should also be able to be specified with the string "NONE" indicating that there
   // are no elements. This allows the option to be present in a config file but left blank.
-  
+
   /*!<\brief addDoubleOption creates a config file parser for an option with the given name whose
    value can be represented by a su2double.*/
-  
+
   void addDoubleOption(const string name, su2double & option_field, su2double default_value) {
     // Check if the key is already in the map. If this fails, it is coder error
     // and not user error, so throw.
     assert(option_map.find(name) == option_map.end());
-    
+
     // Add this option to the list of all the options
     all_options.insert(pair<string, bool>(name, true));
-    
+
     // Create the parser for a su2double option with a reference to the option_field and the desired
     // default value. This will take the string in the config file, convert it to a su2double, and
     // place that su2double in the memory location specified by the reference.
     COptionBase* val = new COptionDouble(name, option_field, default_value);
-    
+
     // Create an association between the option name ("CFL") and the parser generated above.
     // During configuration, the parsing script will get the option name, and use this map
     // to find how to parse that option.
     option_map.insert(pair<string, COptionBase *>(name, val));
   }
-  
+
   void addStringOption(const string name, string & option_field, string default_value) {
     assert(option_map.find(name) == option_map.end());
     all_options.insert(pair<string, bool>(name, true));
     COptionBase* val = new COptionString(name, option_field, default_value);
     option_map.insert(pair<string, COptionBase *>(name, val));
   }
-  
+
   void addIntegerOption(const string name, int & option_field, int default_value) {
     assert(option_map.find(name) == option_map.end());
     all_options.insert(pair<string, bool>(name, true));
     COptionBase* val = new COptionInt(name, option_field, default_value);
     option_map.insert(pair<string, COptionBase *>(name, val));
   }
-  
+
```

Comment:
This is an example of what I meant in my comment: whitespaces in between function declarations/implementations are removed. This causes a lot of problems when it goes into develop and then other branches merge develop in. It also happens on other header files (e.g. geometry_structure.hpp, solver_structure.hpp).

Reply:
@rsanfer, I have no idea how the whitespace changes came in. Probably due to the many merges with develop that have been carried out. I really don't know how to fix this in an automated manner (I am open to suggestions here). If it is really necessary I have to do it manually, but that is not something I am looking for.

Edwin

Reply:
Yes, I can imagine, there were a lot of merges involved over time. I'm not sure how to fix it either in an automated way - when I put the pull request of the adjoint-FSI, I had a similar problem and I fixed it manually (definitely not optimal). I'm sure there have to be ways out there. I have discussed this with @economon before, and we haven't found yet the best way to deal with this. Maybe something we can have a chat on the meeting in a couple of weeks.

Reply:
xxdiff works reasonably well to resolve this. However, it remains a pain.

Reply:
I've used Meld before for the same purpose. But I agree, it's a pain.

Reply:
@rsanfer, I modified the files in Common/include. Could you have a look at them to see if these files correspond better with the current develop version? If so, I'll go through the other directories as well.

Reply:
Thanks @rsanfer. I think I handled most of them, but I am not sure whether I solved the encoding conversion error. If you can have a look, I would appreciate that.

Edwin

Reply:
Indeed, the indentation issues are all sorted, thanks a lot @vdweide. 

I still saw the character encoding conversion error, so I fixed it locally and just committed the changes. I also merged in the current develop with a recent bugfix in PR #572. I cancelled the regression tests before they had started so they run only once. They should be running soon. 

I will have a deeper look on the actual code (outside of these formatting issues) as soon as I am done with some ongoing stuff. Really interested on this new DG-FEM capability.